server/src/main/java/org/elasticsearch/transport/OutboundHandler.java

```diff
@@ -60,8 +65,24 @@
         this.bigArrays = bigArrays;
     }
 
+    void sendPing(TcpChannel channel, BytesReference serializedPing, ActionListener<Void> listener) {
+        MeanMetric bytesMetric;
+        if (channel.isServerChannel()) {
+            bytesMetric = outboundPongsMetric;
+        } else {
+            bytesMetric = outboundPingsMetric;
+        }
+        SendContext sendContext = new SendContext(channel, bytesMetric, () -> serializedPing, listener);
```

Comment:
This can just be deduplicated by adding a `metric` parameter to `sendBytes` and reusing it here. Though (see my other comment on this) we probably don't need to even track the bytes in pings and pongs separate from other bytes.